### ISSUE TITLE
Running Angular Firebase Project with CircleCI

### ISSUE THREAD

**Issue by juancarlosjr97** [angular/angular-cli#14619]
I have my Firebase Angular Project with CircleCI.

But I countered a problem that I need to type to Google Analytics 'YES'

![image](https://user-images.githubusercontent.com/36451129/58734236-c6117a00-83ee-11e9-9752-5697c175b405.png)

I tried to do what the documentation says:

```
ng analytics off

```
Environment Variables 
```
NG_CLI_ANALYTICS = false

NG_CLI_ANALYTICS = False 

NG_CLI_ANALYTICS = ci
```
But they did not work

I found a solution after I read it on https://stackoverflow.com/questions/56355499/stop-angular-cli-asking-for-collecting-analytics-when-i-use-ng-build

```
sudo npm install -g @angular/cli > /dev/null
```

**Comment by alan-agius4** [comment]
Hi, are you using Angular CLI version 8.0.1? As there was a fix CI environments.

**Comment by juancarlosjr97** [comment]
No, I am using ```@angular/cli 7.3.4```

**Comment by alan-agius4** [comment]
This issue should have been solved in version 8.0.1.

If the problem persists after upgrading, please open a new issue, provide a simple repository reproducing the problem, and describe the difference between the expected and current behavior.

**Comment by webcat12345** [comment]
I was using `@angular/cli 8.0.0` and after upgrading to `8.0.1`, issue resolved
